log/log.go

```diff
@@ -59,17 +64,92 @@ type ContextLogger interface {
 
 // init initializes the global logger to sane defaults.
 func init() {
-	Log.Formatter = &logrus.TextFormatter{
-		FullTimestamp: true,
+	switch strings.ToLower(os.Getenv("LOG_FORMAT")) {
+	case "ndjson", "json":
```

Comment:
could we have consts for these?

Reply:
This was just a simple hack to make it easier for me to configure it on Hosted Mender. I didn't mean for it to become a feature.